scripts/insert_into_appointments.sql

```diff
@@ -1 +1,38 @@
 -- insert appointments creating connections between doctors and patients
+
+INSERT INTO appointments(start_at, length, patient_id, doctor_id)
```

Comment:
This won't run since you're trying to insert into the `start_at` and `length` columns, which don't exist.